docs/docsite/rst/community/development_process.rst

```diff
@@ -29,6 +29,57 @@ Ansible accepts code via **pull requests** ("PRs" for short). GitHub provides a
 
 Because Ansible receives many pull requests, we use an automated process to help us through the process of reviewing and merging pull requests. That process is managed by **Ansibullbot**.
 
+Changelogs
+----------
+
+Please add a changelog fragment with any PR that is
+changelog-worthy (features, major bugfixes, backwards
+incompatiblities, etc) except for new modules and plugins (Our tooling
+will automatically add any new module or plugin)
+
+Ansbile developers in the community will require a changelog for any PRs to be
+merged that should appear in the changelog have a changelog fragment.
+This includes new features, major bugfixes, backward incompatibilities,
+deprecations, and other things that users should know on update.
+
+Minor bugfixes that are going to be backported we would want changelog
+entries in the backport PR (if they didn't already appear in the devel
+PR) as our changlog policy is for minor releases to list a short
+summary of all changes.
+
+How to
+~~~~~~
```

Comment:
I'd use "Creating a changelog entry" instead of "How to" (which could be anything) as the header. Also, make the heading match our standards - see https://github.com/ansible/ansible/issues/35520.